addok_france/utils.py

```diff
@@ -50,14 +50,17 @@
 
 
 def clean_query(q):
-    q = re.sub(r'([\d]{5})', r' \1 ', q, flags=re.IGNORECASE)
+    q = re.sub(r'(^| )(boite postale|b\.?p\.?|cs|tsa|cidex) *(n(o|°|) *|)[\d]+ *', r'\1', q, flags=re.IGNORECASE)
```

Comment:
Regex is quite slow, I'm not sure we wanna go that far in cleaning.
Have you had a look on perfs? :)

Reply:
regexp are not that slow (I measured 3µs for this one)... and it is called just once to clean the query